File: SparkLabs/6. Spark - Scala - Data Frames/01.scala

import org.apache.spark.sql.types._
 import org.apache.spark.sql._

val orders= Seq(
Row(1,"DP-203 Azure Data Engineer",10),
Row(2,"AZ-104 Azure Administrator",20),
Row(3,"DP-900 Azure Data Fundamentals",30))

val schema = StructType(Array(
 StructField("id", IntegerType,true),
 StructField("coursename", StringType,true),
 StructField("quantity", IntegerType,true)
))


val rdd = spark.sparkContext.parallelize(orders)
val df=spark.createDataFrame(rdd, schema)


df.show()

File: SparkLabs/3. Spark - Python - Data Frames/01.py
import findspark
findspark.init()
findspark.find()

from pyspark import SparkContext, SparkConf
from pyspark.sql import SparkSession

spark=(
    SparkSession
    .builder
    .appName("DP203")
    .getOrCreate()
)
spark

from pyspark.sql.types import StructType, StructField, StringType, IntegerType

schema = StructType([
    StructField("id", IntegerType(), True),
    StructField("coursename",StringType(), True),
    StructField("quantity", IntegerType(), True)
])

orders= [
(1,"DP-203 Azure Data Engineer",10),
(2,"AZ-104 Azure Administrator",20),
(3,"DP-900 Azure Data Fundamentals",30)]

df = spark.createDataFrame(orders, schema)

df.show()